### ISSUE TITLE
Prefixes with a space don't work

### ISSUE THREAD

**Issue by GamingGeek** [iBlitzkriegi/Vixio#92]
```
discord command test:
    prefixes: %{prefix}%
    trigger:
        reply with "Working!"```
```
```
discord command $prefixtest:
	trigger:
		reply with "test prefix as 'fire '"
		set {prefix} to "fire "
		wait 10 seconds
		reply with "test prefix as 'test '"
		set {prefix} to "test "
```
Execute the command "$prefixtest" and then test the "test" command using the prefix it says.

Expected result: When typing "fire test" and "test test", the bot should reply with "Working!"
Actual result: Nothing happens when you type "fire test" and "test test"

**Comment by Blueyescat** [comment]
https://github.com/iBlitzkriegi/Vixio/issues/61

**Comment by GamingGeek** [comment]
@Blueyescat Yeah, I saw your issue. I got spaces to work by saving it as a variable and using that for the prefix, but since pre-2 it's stopped working

**Comment by Blueyescat** [comment]
Actually your issue is different, mine should be because how the `prefixes` entry is parsed.

**Comment by Pikachu920** [comment]
this is a skript "issue"

**Comment by Blueyescat** [comment]
@Pikachu920 how

**Comment by Pikachu920** [comment]
skript doesn't care about excess spaces

**Comment by Blueyescat** [comment]
The prefixes entry is parsed using a special regex that doesn't come from Skript.
